site/content/en/docs/contrib/external-packages.en.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Managing External Packages"
+linkTitle: "Managing External Packages"
+---
+
+## Managing External Packages
+
+### Chocolatey (Windows)
+
+Chocolatey is a software package manager for Windows that wraps installers, executables, zips, and scripts into compiled packages. Use Chocolatey to install Minikube on Windows. The following links provide details necessary to install, get status, and update to the latest minikube chocolatey package on windows.
+
+* Minikube package on Chocolatey community - [Install, Upgrade and Uninstall](https://community.chocolatey.org/packages/Minikube/).
```

Comment:
Let's add info who maintains this package and is it referring to latest version ?